### PR TITLE
platform services user and policy

### PR DESCRIPTION
**Pull request by Joseph-Irving**

### CODE REVIEW

File: users.tf [hunk 1]
```diff
@@ -37,3 +37,28 @@ resource "aws_iam_group_membership" "admin_group_membership" {
...
+  name = "acp-platform-services-${var.environment}"
+}
+
+resource "aws_iam_group" "platform_services_group" {
+  name = "acp-platform-services-${var.environment}"
+}
+
+resource "aws_iam_policy" "platform_services_policy" {
+  name   = "acp-platform-services-${var.environment}}"
+  policy = "${file("platform_services.json")}"
```
Comment (gambol99) [inline users.tf]: can we use the terraform [native](https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html) implementation .. The reason being that terraform can calculate the diff, while content can only be applied

**Review by vaijab — COMMENTED** [review]
This looks good from a use-case perspective for now. I cannot comment on TF correctness.

**Review by marcinc — APPROVED** [review]
LGTM

File: users.tf [hunk 2]
```diff
@@ -37,3 +37,28 @@ resource "aws_iam_group_membership" "admin_group_membership" {
...
+resource "aws_iam_user" "platform_services_user" {
+  name = "acp-platform-services-${var.environment}"
+}
+
+resource "aws_iam_group" "platform_services_group" {
+  name = "acp-platform-services-${var.environment}"
+}
+
+resource "aws_iam_policy" "platform_services_policy" {
+  name   = "acp-platform-services-${var.environment}}"
```
Comment (gambol99) [inline users.tf]: double '}}'

Comment (Joseph-Irving) [reply users.tf]: I've fixed this

**Review by gambol99 — APPROVED** [review]

**Pull request merged**